packages/gestalt-cli/src/migrate.js

```diff
@@ -82,19 +87,27 @@ async function updateJSONSchema(
 async function updateDatabaseSchema(
   localPackage: Object,
   schemaText: string,
+  {url}
 ): Promise {
-  const {databaseURL} = await get({
-    name: 'databaseURL',
-    message: 'what is the url to your database?',
-    default: `postgres://localhost/${snake(localPackage.name)}`,
-  });
+  let databa;
```

Comment:
Can you leave this as `databaseURL` - I just prefer more descriptive vs abbreviated variable names?